geowebcache/core/src/test/java/org/geowebcache/storage/AbstractBlobStoreTest.java

```diff
@@ -1129,7 +1133,15 @@ protected void assertNoTile(
         TileObject to =
                 TileObject.createQueryTileObject(
                         layerName, new long[] {x, y, z}, gridSetId, format, parameters);
-        assertThat(store.get(to), describedAs("don't get a tile", is(false)));
+        assertNoTile(to);
+    }
+
+    private void assertNoTile(TileObject to) {
+        await().atMost(5, SECONDS) // give stores with async deletes a chance to complete
+                .untilAsserted(
+                        () ->
+                                assertThat(
+                                        store.get(to), describedAs("don't get a tile", is(false))));
```

Comment:
Note I had to use `Awaitlity` for the tests in `AbstractBlobStoreTest` that do deletes because the online tests may be too slow to complete async deletes on Azure. Couldn't make them fail on my dev machine, but they failed randomly on github actions

Reply:
No problem.